pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -36,38 +36,54 @@ protected AggregatedNamespaceStats initialValue() throws Exception {
         }
     };
 
+    private static FastThreadLocal<TopicStats> localTopicStats = new FastThreadLocal<TopicStats>() {
+        @Override
+        protected TopicStats initialValue() throws Exception {
+            return new TopicStats();
+        }
+    };
+
     public static void generate(PulsarService pulsar, boolean includeTopicMetrics, SimpleTextOutputStream stream) {
         String cluster = pulsar.getConfiguration().getClusterName();
         AggregatedNamespaceStats namespaceStats = localNamespaceStats.get();
-        TopicStats topicStats = new TopicStats();
+        TopicStats topicStats = localTopicStats.get();
 
         pulsar.getBrokerService().getMultiLayerTopicMap().forEach((namespace, bundlesMap) -> {
             namespaceStats.reset();
 
             bundlesMap.forEach((bundle, topicsMap) -> {
                 topicsMap.forEach((name, topic) -> {
                     getTopicStats(topic, topicStats);
-                    namespaceStats.updateStats(topicStats);
+
```

Comment:
Don't understand the reasoning for 

namespaceStats.updateStats(topicStats);  only if includeTopicMetrics == false